install/uninstalling_cluster.adoc

```diff
@@ -31,6 +31,31 @@ installed by Ansible, including:
 The playbook deletes content for any hosts defined in the inventory file
 that you specify when running the playbook.
 
+[IMPORTANT]
+====
+Before you uninstall your cluster, make sure that uninstalling is the best
+option.
+
+* If your installation process failed and you want to continue the process,
+you can
+xref:../install/running_install.adoc#advanced-retrying-installation[retry the installation].
+The installation playbooks are designed so that if they fail to install your
+cluster, you can run them again without needing to uninstall the cluster.
+* If you want to restart a failed installation from the beginning, you can
```

Comment:
I don't understand how this is different from the default uninstall procedure in the immediately following section? Do you run the uninstall without the `-i /path/to/file` so that no inventory file is used?

I suspect our customers, who are more familiar with the 3.x method, will be able to understand the differences between this and the default uninstall, but it might make sense to spell them out here.

Reply:
I think this was meant to specify that it's ok to use the uninstall playbook IFF the customer experienced a failed installation and wants to start the installation over using the exact same setup, minor/asynchronous versions, etc. This is really the only time the uninstall playbook is ok to run (imvho)

Reply:
Yes. I'm going to add "as described in the following section" to try to clarify this part. Thanks!